composer.json

```
@@ -46,6 +46,7 @@
         "symfony/property-access": "^5.4 || ^6.2",
         "symfony/property-info": "^5.4 || ^6.2",
         "symfony/routing": "^5.4 || ^6.2",
+        "symfony/runtime": "^6.3",
```

```suggestion
        "symfony/runtime": "^5.4 || ^6.2",
```
like others dependencies